build/builder.sh

```diff
@@ -119,6 +119,9 @@ vols="${vols} --volume=${gopath0}/bin/docker_amd64:/go/bin"
 vols="${vols} --volume=${HOME}/.jspm:${container_home}/.jspm"
 vols="${vols} --volume=${HOME}/.npm:${container_home}/.npm"
 vols="${vols} --volume=${cockroach_toplevel}:/go/src/github.com/cockroachdb/cockroach"
+# Add ssh key for terraform tests.
+vols="${vols} --volume=${HOME}/.ssh/google_compute_engine.pub:${container_home}/.ssh/google_compute_engine.pub"
```

Comment:
I'm pretty sure this is not needed or used. On my machine, this file doesn't exist, and I've run the terraform tests.